apps/uci/web/src/components/GetBotList.ts

```diff
@@ -0,0 +1,83 @@
+import { getBotDetailsList } from "@/utils/api-handler";
+import { normalizeUsers } from "@/utils/normalize-user";
+import { reverse, sortBy, without } from "lodash";
+import moment from "moment";
+import { useEffect } from "react";
+import { toast } from "react-hot-toast";
+
+interface GetBotItemProp{
+    setUsers: React.Dispatch<React.SetStateAction<any>>;
+    setCurrentUser: React.Dispatch<React.SetStateAction<any>>;
+    setLoading: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Since all the types are same so you can define a separate type for this and re use it